tox.ini

```diff
@@ -102,16 +102,25 @@ tags =
     unit
 
 [testenv:py27-lint]
+deps = 
+    -rlint-requirements.txt
+skip_install = true
```

Comment:
Unfortunately, on travis the job is still 3 minutes instead of 5 seconds, the requirements are still installed somehow ... So it's a first step but not sufficient I am afraid.

Reply:
Try `usedevelop = False`